templates/webapi/main/group_builds_functionality_view.html.ep

```diff
@@ -0,0 +1,58 @@
+% for my $child (@{$children}) {
+  % my $has_jobs = 0;
+  % my $title_class_mod = "text-muted";
+
+  % for my $build_res (@$build_results) {
+    % my $child_res = $build_res->{children}->{$child->{id}};
+    % my $jobs = $child_res->{passed} + $child_res->{unfinished} + $child_res->{softfailed} + $child_res->{failed} + $child_res->{skipped};
+
+    % if ($jobs) {
+      % $has_jobs = 1;
+      % $title_class_mod = "";
+    % }
+  % }
+
+  <div class="mb-2">
+    <div class="h4 <%= $title_class_mod %> mb-2">
+      %= link_to $child->{name} => url_for('group_overview', groupid => $child->{id})
+    </div>
+
+    % for my $build_res (@$build_results) {
+      % my $build = $build_res->{build};
+      % my $child_res = $build_res->{children}->{$child->{id}};
+      % my $jobs = $child_res->{passed} + $child_res->{unfinished} + $child_res->{softfailed} + $child_res->{failed} + $child_res->{skipped};
```

Comment:
I still think this computation should not be repeated here.

Reply:
That's actually unchanged from before, see templates/webapi/main/group_builds.html.ep . I agree in principle though and I think we can simply use the value "total" here. Changed.